shared/redux/rtk-apis/users/users.api.ts

```diff
@@ -12,8 +12,17 @@ const usersApi = projectApi.injectEndpoints({
       query: () => "users/user-details",
       providesTags: ["Profiles"],
     }),
+
+    editUser: builder.mutation<IUser, EditableUserFields>({
+      query: (editUserData) => ({
+        url: "users/edit",
```

Comment:
Can use the same route as the get here
```suggestion
        url: "users/user-details",
```